src/main/java/org/truffleruby/core/hash/library/PackedHashStoreLibrary.java

```diff
@@ -63,6 +63,12 @@ public final class PackedHashStoreLibrary {
     private static final int ELEMENTS_PER_ENTRY = 3;
     public static final int TOTAL_ELEMENTS = MAX_ENTRIES * ELEMENTS_PER_ENTRY;
 
+    private static final boolean bigHashTypeIsCompact;
+    static {
+        String hashType = System.getProperty("BigHashStrategy");
+        bigHashTypeIsCompact = hashType != null && hashType.equalsIgnoreCase("compact");
+    }
```

Comment:
Could you update this so it defaults to your new CompactHashStore? (i.e. use it unless `BigHashStrategy == "buckets"`
Then we'll be sure the new strategy passes the whole CI.

Reply:
The same logic is duplicated in src/main/java/org/truffleruby/core/hash/HashLiteralNode.java, could you deduplicate?